Assets/Editor/E05_Controls.cs

```diff
@@ -77,6 +101,9 @@ void AddTask()
             {
                 m_TasksContainer.contentView.AddChild(CreateTask(m_TextField.text));
                 m_TextField.text = "";
+
+                // Give focus back to text field.
+                m_TasksContainer.focusController.SwitchFocus(m_TextField);
```

Comment:
Not sure to understand what you mean here. There is one focus controller per panel. myVisualElement.focusControler is a shortcut for myVisualElement.panel.focusController.